### DISCUSSION TITLE
滑手機要不要開「暗黑模式」？三好二壞告訴你

### DISCUSSION THREAD

**Discussion by HuangHsinTzu** [1101-GS4719-C/Python-Community-GS4719-C#1067] [General]
我自己手機就是開暗黑模式
但我單純只是覺得黑底白字比較好看哈哈
可能也有覺得對眼睛比較好吧
好像很多人都覺得開暗黑模式對眼睛比較好

文章提到其實人眼的機制較習慣在亮光下觀察暗色，而非反過來在暗黑的背景裡閱讀亮色
而且黑底的環境下虹膜需要擴張
長期使用案黑模式，眼睛的肌群其實更容易疲勞

有人可能會說可是正常模式的藍光不是會比較多嗎
可能真的多那麼一點點吧
使用暗黑模式同樣也有藍光
想減少藍光的吸收最直接的方就法是少用3C

不過使用案黑模式最大的好處就是省電!

以上的pros and cons分析 希望可以幫助大家決定要不要開暗黑模式
[點我看更詳細內容](https://3c.ltn.com.tw/news/38491)

**Comment by Zoe06060606** [comment]
我超喜歡暗黑模式只要應用程式能修改的，我都改成深色模式了，白色有時候看起來真的偏刺眼

**Comment by Jasper525** [comment]
我覺得關鍵是環境光源，像我不開檯燈看黑底白字，比開檯燈看白底黑字，眼睛會更酸。

**Comment by Valentinaa0214** [comment]
我自己比較習慣白底黑字，黑底對我來說超吃力的，
關燈就不要繼續看手機了，感覺對眼睛超不好的。

**Comment by ghost** [comment]
看完文章後，雖然我了解了因為黑底的環境需要虹膜擴張，所以眼睛的肌群其實會更容易疲勞，的這個事實，但我認為實際上並不能認為黑底就真的對眼睛傷害較高，因為它減少了螢幕的發光面積，雖虹膜擴張導致吸收光的面積增加，但射向眼睛的光亮度卻減少了，這一來一回，究竟是誰對眼睛的傷害更高就說不準了。所以還是得看網站顏色分配和螢幕的發光性來決定他比較適合哪種底。

**Comment by s96015welson** [comment]
我自己的手機是Amoled螢幕，所以改成黑底白字的時候感覺上對於舒緩眼部疲勞有非常大的幫助，因為螢幕黑色部分是完全不發光的，而白色底就很刺眼。不過除了使用黑暗模式，定時使眼睛休息、避免環境光源太暗等等也仍然很重要。

**Comment by peggy0329** [comment]
我在白天時習慣使用白底黑字，但在暗黑處就比較習慣黑底白字。主要原因應該也是背景跟光源的相似度，白天光源佳習慣較明亮的白底中看字，光源不佳時暗黑模式可以減少我的眼睛疲勞。我認為這可能也跟我平常有戴角膜塑型片的關係有關，至從戴了後，眼睛很容易疲勞，無法長時間在暗的地方看手機。

**Comment by AnAnistired** [comment]
其實我覺得在暗處用暗黑模式眼睛會比用原本的還要舒服欸，當然在暗處滑手機本來就會讓眼睛變得很疲勞，而保護眼睛更好的方法就是少用手機等3C😂

**Comment by Twenty0522** [comment]
在光源不足的環境下使用手機，即使將螢幕亮度調到最低，有時候眼睛還是會覺得有點刺激，因此暗黑模式幫助我在閱讀時不感到刺眼，我覺得是很貼心的設計。題外話：看了文章連結才發現暗黑模式竟然可以省電43%，獲得一個實用的小知識了。

**Comment by wenchi9023** [comment]
感覺暗黑模式只是讓使用者在強光下增加可視性
我覺得不代表在光源不足時使用暗黑模式就比較護眼吧(?)
我反倒好奇夜間模式和暗黑模式的功能是否相同?

**Comment by ru0372572** [comment]
我也是習慣白底黑字，但身邊的人很多都用黑底，也說用平板寫筆記時比較好寫，字還會比較好看(?
看來省電就是暗黑模式最大的優點了呢，因為在低光環境下暗黑模式只是能讓我們閱讀的更舒適，而沒有保護眼睛的功用，總之~還是在光線充足的地方使用手機吧

**Comment by Sylvie0601** [comment]
我手機都是白光加護眼模式，自認為自己還蠻常滑手機的哈哈，不知道開護眼模式對眼睛有沒有比較好哈哈哈

**Comment by yuxinlin93** [comment]
我比較喜歡深色模式，所有東西幾乎都切成深色模式了，連桌布跟鎖定螢幕都是黑色系的，但不是為了平常滑手機才切的，是因為平常很常在暗暗的環境下用手機(對 其實對眼睛很不好) 如果是淺色模式眼睛會很不舒服，尤其是睡覺起來開手機看時間的時候，如果是淺色模式，我眼睛會直接被閃瞎

**Comment by a45320614** [comment]
對我來說深色模式看起來比較舒適，所有我把能改深色模式的應用程式都改了，要閱讀白底的網站時就會把螢幕亮度調低一點，希望大家都能保護好自己的靈魂之窗，不要讓厚重的眼鏡伴隨整個人生。

**Comment by orianna901001** [comment]
我的天這跟我想的好不一樣哦，我自己通常手機亮度都不可能切超過三分之一，除非是在室外才會全開，原來我的眼睛都經歷了這些，果然避免近視還是少用3c好了QQ

**Comment by llh26** [comment]
我完全是個深色模式愛好者，自從發現可以換成深色模式後，就回不去了～我的眼睛本來就有點畏光，平常螢幕都調超暗（對我來說還好），別人都懷疑我到底有沒有開機，通常超過一半的亮度我的眼睛就受不了了，所以暗色模式真的是我的救星，沒想到還有這些壞處，但我還是覺得深色模式拯救了我被閃瞎、眼睛痠痛的問題，目前覺得淺色模式對我而言弊大於利。

**Comment by ziihui** [comment]
黑底模式真的超讚~相對於白底模式，黑底模式可以讓螢幕亮度降低，反射也減少，尤其是在暗光環境下，對眼睛的負擔也更小。
不但可以讓閱讀更順暢，也能保護眼睛健康。另外，黑底模式也能減少藍光的傷害，因為藍光會干擾我們的睡眠。
最後，黑底模式還能省電和減少眩光，讓我們的上網體驗更棒!

**Comment by Jamesmoneya** [comment]
應該還是會選擇暗黑模式，因為現在看回去白底 真的太亮了，但一下就會覺得很刺眼，而且我覺得暗黑跟白底模式下，暗黑模式會讓整個介面看起來非常好看，就是會有比較有質感比較高級的感覺，一邊滑的過程中，也是讓自己更沉醉在手機的世界中

**Comment by KevinCheng42** [comment]
雖然暗黑模式看起來很新潮，但我還是習慣用白底模式，因為我通常都開著燈使用3C產品。但是，如果是在晚上時在戶外用手機，剛打開時的確會覺得很刺眼～

**Comment by ashleyy-huang** [comment]
自從手機有暗黑模式之後我根本回不去白底模式了，他對於我的眼睛使用較不刺激，也大大的節省手機電量，不會再那麼容易沒電。我想比起暗黑和白底模式，最重要的是使用手機時環境的亮度，我們應該要避免在黑暗的情況下盯著螢幕，減少對眼睛的負荷。

**Comment by brian431** [comment]
在用電腦的時候畫面一片白有時候的確是會覺得有些刺眼，因此個人也是習慣使用暗黑模式。不過在用電腦的時候有些網站沒辦法用暗黑模式，突然從一片黑變一片白實在很不爽。

**Comment by CCello** [comment]
我個人認為黑底白字比較好看，不過這是一個個人喜好和視覺體驗的問題，不同人對於不同配色和字型的喜好不同，所以開啟暗黑模式是否好看，需要根據個人的感受來決定。至於暗黑模式是否對眼睛比較好是一個爭議性的問題。事實上，眼睛對於亮度和對比度的敏感度是因人而異的，有些人可能會感到在暗黑背景下閱讀亮色文字比較舒適，而有些人則相反。對於長時間使用暗黑模式是否會造成眼睛疲勞，研究結果並不一致。有些研究顯示，在低光環境下使用暗黑模式可以減少眼睛疲勞，但也有一些研究認為，長時間使用暗黑模式可能會導致眼睛更容易疲勞。

**Comment by 111607004** [comment]
我覺得環境光對螢幕的顏色有很大的影響，如果周圍都亮亮的，用深色模式看起來就會蠻吃力的，但如果是晚上在外面，或是宿舍都關燈的時候，用正常模式就好像眼睛直接對著日光燈管一樣，超級刺眼。不過還是盡量不要在都沒燈的情況下用手機啦，就算是深色模式稍微看一下也會覺得睛刺痛不舒服。

**Comment by Elaine2899** [comment]
我也是能改暗黑模式的都改了 感覺比較不會那麼刺眼，尤其是晚上睡前突然想確認一下有沒有開鬧鐘的時候，沒開暗黑模式真的覺得眼睛快瞎了

**Comment by bestbanana295** [comment]
感覺如果是在黑暗的環境底下的話就真的不要再使用3C產品了，如果在黑暗中使用3C，會加快眼睛老化。眼睛是靈魂之窗必須好好照顧才行。
